S21/rkarmuri/ConvertToLowerCase.java

```diff
@@ -0,0 +1,98 @@
+//Method to convert given string to lowercase
+package com.company;
+
+import java.util.Locale;//Package to accept multiple languages
+
+public class ConvertToLowerCase {
+    public String convert(String wordSample, String language) {
+        System.out.println("Input word: " + wordSample);
+        System.out.println("Input language tag: " + language);
+        String wordLowerCase = wordSample.toLowerCase();//Conversion in English
+        String trazString = wordSample.toLowerCase(Locale.forLanguageTag("az"));//Conversion in Turkish and Azerbaijani
+        String russString = wordSample.toLowerCase(Locale.forLanguageTag("ru-Latn"));//Conversion in Russian
+        String hungarianString = wordSample.toLowerCase(Locale.forLanguageTag("hu"));//Conversion in Hungarian
```

Comment:
Can we combine all this variables? For example:

String worlLowerCase = wordSample.toLowerCase(Locale.forLanguageTag(language));

Reply:
We can do that, but I wanted these variables for each of these exceptional languages to be used write the conditions.